server/src/main/java/fr/uga/l3miage/tp1/exo3/models/AssociationEntity.java

```diff
@@ -0,0 +1,22 @@
+package fr.uga.l3miage.tp1.exo3.models;
+
+import fr.uga.l3miage.tp1.exo3.enums.AssociationType;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
+@Entity
+public class AssociationEntity {
+    @Id
+    private String name;
+    @NotNull
+    private String presidentName;
+    @NotNull
+    private String treasureName;
+    private String secretaryName;
+    @Enumerated(EnumType.STRING)
```

Comment:
Si je ne précise pas de type, c'est le type par défaut, donc la c'est plutôt : 
```suggestion
    @Enumerated(EnumType.ORDINAL)
```